### PR TITLE
add(epp): applications-cisco-ssms-restapi FR

### PR DESCRIPTION
**Pull request by psamecentreon**

### CODE REVIEW

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+id: applications-cisco-ssms-restapi
+title: Cisco SSMS
+---
+
+## Vue d'ensemble
+
+Cisco Smart Software Manager On-Prem (anciennement connu sous le nom de Cisco Smart Software Manager satellite) est un composant de Cisco Smart Licensing
+qui fonctionne en conjonction avec Cisco Smart Software Manager (SSM). 
+Il offre une visibilité et des rapports en temps quasi réel sur les licences Cisco que vous achetez et consommez,
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:10]:
```suggestion
Il offre une visibilité et des rapports sur les licences Cisco que vous achetez et consommez,
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Alerts-->
+
+| Metric Name                | Description              |
+| :------------------------- | :------------------------|
+| account.alerts.minor.count | Counter of alerts minor. |
+| account.alerts.major.count | Counter of alerts major. |
+
+
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:34]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+<!--Licenses-->
+
+| Metric Name                | Description                                        |
+| :------------------------- | :------------------------------------------------- |
+| licenses.usage.count       | Counter of licenses usage. Unit : Bytes (B)        |
+| licenses.free.count        | Counter of licenses free. Unit : Bytes (B)         |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:40]: Wrong unit

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+| licenses.usage.percentage  | Percentage of licenses usage. Unit : percentage(%) |
+
+
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:43]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+### Privilèges API
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:48]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+Un compte de service est requis pour interroger l'API Cisco SSMS. Celui-ci doit avoir suffisamment de privilèges en lecture du compte ciblées.
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:50]: Maybe add a link to the official documentation.

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+2. Sur le serveur Central Centreon, installer le Plugin-Pack via le RPM:
+
+```bash
+yum install centreon-pack-applications-cisco-ssms-restapi.noarch
+```
+
+3. Sur l'interface Web de Centreon, installer le Plugin-Pack *Cisco SSMS* depuis la page "Configuration > Plugin packs > Manager"
+
+
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:82]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+```bash
+yum install centreon-plugin-Applications-Cisco-Ssms-Restapi.noarch
+```
+
+2. Sur le serveur Central Centreon, installer le Plugin-Pack via le RPM:
+
+```bash
+yum install centreon-pack-applications-cisco-ssms-restapi.noarch
+```
+
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:79]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+3. Sur l'interface Web de Centreon, installer le Plugin-Pack *Cisco SSMS* depuis la page "Configuration > Plugin packs > Manager"
+
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:87]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+
+Ce Plugin-Pack est concçu de manière à avoir dans Centreon un hôte par compte
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:89]:
```suggestion
Ce Plugin-Pack est conçu de manière à avoir dans Centreon un hôte par compte.
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+Lorsque vous ajoutez un hôte à Centreon, appliquez-lui le modèle *App-Cisco-Ssms-Restapi-custom*. Une fois celui-ci configuré, certaines macros doivent être renseignées:
+
+| Mandatory   | Nom                    | Description                                                                |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:92]:
```suggestion
| Mandatory   | Name                    | Description                                                                |
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+| :---------- | :--------------------- | :------------------------------------------------------------------------- |
+| X           | APIPORT                | Port used. Default is 443                                                  |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:94]:
```suggestion
| X           | APIPORT                | Port used. Default is 8443                                                  |
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+| X           | APIPROTO               | Protocol used. Default is https                                            |
+| X           | CLIENTID               | Client ID to access to the API.                                            |
+| X           | CLIENTSECRET           | Client Secret to access to the API.                                        |
+|             | APIEXTRAOPTIONS        | Any extra option you may want to add to the command                        |
+
+
+## FAQ
+
+### Comment tester un contrôle en ligne de commandes et que signifient les options principales ?
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:103]:
```suggestion
### Comment puis-je tester le Plugin et que signifient les options des commandes ?
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 14]
```diff
@@ -0,0 +1,136 @@
...
+
+Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commandes depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:105]:
```suggestion
Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commande depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 15]
```diff
@@ -0,0 +1,136 @@
...
+
+```bash
+/usr/lib/centreon/plugins/centreon_cisco_ssms_restapi.pl \
+    --plugin=apps::cisco::ssms::restapi::plugin \
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:109]:
```suggestion
      --plugin=apps::cisco::ssms::restapi::plugin \
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 16]
```diff
@@ -0,0 +1,136 @@
...
+	--mode=licenses \
+	--hostname='myipaddress' \
+	--client-id='myapiclientid' \
+	--client-secret='myapiclientsecret' \
+	--account='1234abc-56de-78fg-90hi-1234abcdefg' \
+	--filter-counters='status' \
+	--filter-license-name='mylicence'
+	--critical-license-status='%{status} !~ /in compliance/i' \
+	--verbose
+
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:119]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+	--client-secret='myapiclientsecret' \
+	--account='1234abc-56de-78fg-90hi-1234abcdefg' \
+	--filter-counters='status' \
+	--filter-license-name='mylicence'
+	--critical-license-status='%{status} !~ /in compliance/i' \
+	--verbose
+	
+```
+
+La commande ci-dessus contrôle le statut  des licences Cisco SSMS (```--mode=licences```) nommée *mylicence* (```--filter-licences-name='mylicence'```).
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:122]:
```suggestion
La commande ci-dessus contrôle le statut des licences Cisco SSMS (```--mode=licences```) nommées *mylicence* (```--filter-licences-name='mylicence'```).
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 18]
```diff
@@ -0,0 +1,136 @@
...
+Cette licence appartient au compte *1234abc-56de-78fg-90hi-1234abcdefg* (```--account='1234abc-56de-78fg-90hi-1234abcdefg'```). 
+
+Cette commande déclenchera une alarme CRITICAL s'il contient le mot *in compliance*  (```--critical-license-status='%{status} !~ /in compliance/i'```).
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:125]:
```suggestion
Cette commande déclenchera une alarme CRITICAL si le statut de la licence est différent de *in compliance* (```--critical-license-status='%{status} !~ /in compliance/i'```).
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 19]
```diff
@@ -0,0 +1,136 @@
...
+
+
+Toutes les options et leur utilisation peuvent être consultées avec le paramètre ```--help``` ajouté à la commande:
+
+```/usr/lib/centreon/plugins/centreon_cisco_ssms_restapi.pl --plugin=apps::cisco::ssms::restapi::plugin --mode=licences --help```
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:130]:
```suggestion
```
/usr/lib/centreon/plugins/centreon_cisco_ssms_restapi.pl --plugin=apps::cisco::ssms::restapi::plugin --mode=licences --help
```
```

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 20]
```diff
@@ -0,0 +1,136 @@
...
+
+
+### Comment puis-je supprimer les perfdatas *count* dans le cas où je ne souhaite vérifier qu'une seule application ?
+
+Le Plugin permet de filtrer sur un ou plusieurs éléments mais permet également de récupérer l'ensemble des éléments si aucun filtre n'est spécifié.
+De ce fait, des perfdatas "globales" sur les statistiques des objets sont ajoutées par défaut. Il est possible de supprimer ces données de performance en appliquant le filtre suivant: ```--filter-perfdata='^$'```.
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:136]: Maybe add proxy troubleshooting or http curl backend ;-)

**Review by pkriko — CHANGES_REQUESTED** [review]

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md (hunk 6 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:46]: Link added.

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md (hunk 20 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:148]: Troubleshooting added

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md (hunk 3 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md]: Done

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md [hunk 21]
```diff
@@ -0,0 +1,148 @@
...
+
+### J'obtiens le message d'erreur suivant: 
+
+#### ```UNKNOWN: 500 Can't connect to api.meraki.com:443 |```
+
+Lors du déploiement de mes contrôles, j'obtiens le message suivant ```UNKNOWN: 500 Can't connect to api.meraki.com:443 |```.
+
+Cela signifie que Centreon n'a pas réussi à se connecter à l'API Cisco Meraki (*api.meraki.com*).
+
+La plupart du temps, il faut préciser le proxy à utiliser pour requêter l'URL *api.meraki.com* en utilisant l'option ```--proxyurl='http://proxy.mycompany:8080'```.
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md:135]: Remove api.meraki.com from your copy/paste :p

**Review by pkriko — CHANGES_REQUESTED** [review]

File: fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md (hunk 21 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md]: api.meraki.com ref removed

**Review by pkriko — APPROVED** [review]

**Pull request merged**